web/pandas/community/blog/2019-user-survey.md

```diff
@@ -0,0 +1,172 @@
+Title: 2019 pandas user survey
+Date: 2019-08-22
+
+<style type="text/css">
+table td {
+    background: none;
+}
+
+table tr.even td {
+    background: none;
+}
+
+table {
+	text-shadow: none;
+}
+
+</style>
+
+# 2019 pandas user survey
+
+Pandas recently conducted a user survey to help guide future development.
+Thanks to everyone who participated! This post presents the high-level results.
+
+This analysis and the raw data can be found [on GitHub](https://github.com/pandas-dev/pandas-user-surveys) and run on Binder
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/pandas-dev/pandas-user-surveys/master?filepath=2019.ipynb)
+
+
+We had about 1250 repsonses over the 15 days we ran the survey in the summer of 2019.
+
+## About the Respondents
+
+There was a fair amount of representation across pandas experience and frequeny of use, though the majority of respondents are on the more experienced side.
+
+
+
+![png]({{ base_url }}/static/img/blog/2019-user-survey/2019_4_0.png)
+
+
+
+
+![png]({{ base_url }}/static/img/blog/2019-user-survey/2019_5_0.png)
+
+
+We included a few questions that were also asked in the [Python Developers Survey](https://www.jetbrains.com/research/python-developers-survey-2018/) so we could compare Pandas' population to Python's. 
+
+90% of our respondents use Python as a primary language (compared with 84% from the PSF survey).
+
+
+
+
+
+    Yes    90.67%
+    No      9.33%
+    Name: Is Python your main language?, dtype: object
+
+
+
+Windows users are well represented (see [Steve Dower's talk](https://www.youtube.com/watch?v=uoI57uMdDD4) on this topic).
+
+
+
+
+
+    Linux      61.57%
+    Windows    60.21%
+    MacOS      42.75%
```

Comment:
I *think* something went wrong, this doesn't adds up to ```100%```

Reply:
It was a select multiple, so people could pick more than one.